.github/workflows/tests.yaml

```diff
@@ -16,5 +17,4 @@ jobs:
           cache-dependency-path: package-lock.json
       - run: npm ci
       - run: npm run format
```

Comment:
**Requested Change:** This happens in linting, so I think we can remove it from the tests job.

Reply:
Sounds good!